auth/auth/auth.py

```diff
@@ -431,6 +545,35 @@ async def delete_user(request, userdata):  # pylint: disable=unused-argument
     return web.HTTPFound(deploy_config.external_url('auth', '/users'))
 
 
+@routes.delete('/api/v1alpha/user/{user}')
```

Comment:
This is the correct way to write an API endpoint. The billing projects API endpoint should never have been a POST. POSTs should only be used for the web endpoints so that a web form can be used (which only supports GET and POST for silly historical reasons).